src/ContainerLogs.jsx

```diff
@@ -80,11 +80,14 @@ class ContainerLogs extends React.Component {
     }
 
     resize(width) {
+        if (!this.term?._core?._renderService?.dimensions)
```

Comment:
This added line is not executed by any test.

Reply:
This is a bit akward as this now depends on internals. Which I see, it already did...